### ISSUE TITLE
why assign embed.weight to G.ndata['feat'] and inputs at the same time in 1_first.py?

### ISSUE THREAD

**Issue by Jubird915** [dmlc/dgl#2503]
embed = nn.Embedding(34, 5)  # 34 nodes with embedding dim equal to 5
G.ndata['feat'] = embed.weight

inputs = embed.weight

acturally these two values keep the same in one epoch. I was confused.

**Comment by jermainewang** [comment]
It's just to demonstrate that you could save an embedding weight to `ndata`.
